teos/src/main.rs

```diff
@@ -77,14 +82,19 @@ async fn main() {
     });
 
     // Load conf (from file or defaults) and patch it with the command line parameters received (if any)
-    let mut conf = config::from_file::<Config>(path.join("teos.toml"));
+    let mut conf = config::from_file::<Config>(conf_file_path);
     let is_default = conf.is_default();
     conf.patch_with_options(opt);
     conf.verify().unwrap_or_else(|e| {
         eprintln!("{e}");
         std::process::exit(1);
     });
 
+    // Log config file path and any non-default options
+    log::info!("Using configuration file: {:?}", &cloned_conf_file_path);
```

Comment:
Missing items as per the requirements of issue #202 -
- [ ] Logging the default data directory

Reply:
> Missing items as per the requirements of issue #202 -
> 
> * [ ]  Logging the default data directory

Added the default data dir now and working on the network dir that we are currently using.